sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="sandboxed-containers-1-3-release-notes"]
+= {sandboxed-containers-first} {sandboxed-containers-version} release notes
+include::_attributes/common-attributes.adoc[]
+:context: sandboxed-containers-release-notes
+
+toc::[]
+
+[id="sandboxed-containers-1-3-about-this-release"]
+== About this release
+
+These release notes track the development of {sandboxed-containers-first} {sandboxed-containers-version} alongside Red Hat {product-title} {product-version}.
+
+This product is fully supported and enabled by default as of {product-title} 4.10.
+
+[id="sandboxed-containers-1-3-new-features-and-enhancements"]
+== New features and enhancements
+
+
+
+[id="sandboxed-containers-1-3-bug-fixes"]
+== Bug fixes
+
+
+
+[id="sandboxed-containers-1-3-known-issues"]
+== Known issues
+
+
+
+[id="sandboxed-containers-1-3-asynchronous-errata-updates"]
+== Asynchronous errata updates
+
+Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata are available on the https://access.redhat.com/downloads/content/290/[Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.
```

Comment:
For xrefs, please put the introductory phrase first, and then the xref itself: "For more information about asynchronous errata, see the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle].

Reply:
Hi Lisa. All of this content was directly copied from the errata section in the OCP release notes template.